README.md

```diff
@@ -110,6 +110,8 @@ We set this in the [`.env`](https://github.com/openaustralia/planningalerts/blob
 COUNCILLORS_ENABLED=true
 ```
 
+Similarly, you can toggle the availability of the contributing suggested councillors feature on or off for the entire site with the environment variable `CONTRIBUTE_COUNCILLORS_ENABLED`.
```

Comment:
Nice simple wording. Do you think that this section deserves a heading, and a short explanation of what the feature is, like the general councillors section or the donation section, and then the explanation of how to turn it on and off?

When I read this, I wonder what I should set the environment variable to be. Should it be `true` like the `COUNCILLORS_ENABLED` one? Consistency with this stuff is important, they should all work the same way and be explained consistently.